python/graphstorm/gpartition/convert_to_graphbolt.py

```diff
@@ -0,0 +1,75 @@
+"""
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Convert existing partitioned data to GraphBolt format.
+"""
+
+import argparse
+import importlib.metadata
+import logging
+import os
+import time
+
+from dgl import distributed as dgl_distributed
+from packaging import version
+
+
+def parse_gbconv_args() -> argparse.Namespace:
+    """Parses GraphBolt conversion arguments"""
+    parser = argparse.ArgumentParser("Convert partitioned DGL graph to GraphBolt format.")
+    parser.add_argument("--input-path", type=str, required=True,
+                           help="Path to input DGL partitioned data.")
+    parser.add_argument("--metadata-filename", type=str, default="metadata.json",
```

Comment:
Is the graph partition config file <graph_name>.json?

Reply:
When gconstruct creates it yes, when dist_part creates it's metadata.json